config/kubernetes/sig-docs/teams.yaml

```diff
@@ -149,13 +149,15 @@ teams:
     - claudiajkang
     - gochist
     - ianychoi
+    - seokho-son
     privacy: closed
   sig-docs-ko-reviews:
     description: Reviews for Korean docs PRs
     members:
     - ClaudiaJKang
     - gochist
     - ianychoi
+    - seokho-son
     privacy: closed
   sig-docs-l10n-admins:
```

Comment:
@gochist 👋 Do you want to add @seokho-son to `sig-docs-l10n-admins` as well, for branch management purposes? Otherwise LGTM

Reply:
I agree with Zach - @gochist if you are fine, would you kindly add him and me to `sig-docs-l10n-admins` as well?